Add info about Font-Overview + refactor 'Reference' page and dependencies (#92)
googlefonts/fontra-docs: _reference/font-overview/panels/navigation.md

---
title     : Navigation Font Overview
layout    : default
permalink : /reference/font-overview/panels/navigation
draft     : true
order     : 4111
---

<nav aria-label="breadcrumb">
  <ol class="breadcrumb small">
    <li class="breadcrumb-item"><a href="{{ site.url }}">Index</a></li>
    <li class="breadcrumb-item"><a href="{{ site.url }}/reference">Reference</a></li>
    <li class="breadcrumb-item active" aria-current="page">{{ page.title }}</li>
  </ol>
</nav>

<div class='row'>
<div class='col-md' markdown='1'>
![]({{ site.url }}/images/font-overview-navigation.png){: .img-fluid .py-2 }
</div>
<div class='col-md' markdown='1'>

##### Search Glyphs

Search for glyphs containing the entered text, for example *grave* shows *Agrave*, *Egrave*, *Igrave*, etc.

- - -

##### Location

Dropdown
: switch between the specified **font sources**

Slider(s)
: use the slider to navigate continuously through the location

- - -

##### Group by

Sort the glyphs of the font by one or many of the following categories.

Script

> : sort by **script** (a script is not a language: Cyrillic ≠ Bulgarien) 

Reviewer: ```suggestion
: Group by **script** (a script is not a language: Cyrillic ≠ Bulgarian) 
```

Block

> : sort by **unicode blocks**

Reviewer: ```suggestion
: Group by **unicode block**
```

Case

> : sort by **uppercase** (followed by *lowercase*) glyphs

Reviewer: ```suggestion
: Group by **case**, organizing in upper-, lower- and minor-case
```

Category

> : sort by **category**, for example: Letter, Number, Punctuation, etc.

Reviewer: ```suggestion
: Group by **category**, for example: Letter, Number, Punctuation, etc.
```

Sub-category

> : sort by **sub-category**, for example: Superscript, Fraction, Quote, etc.

Reviewer: ```suggestion
: Group by **sub-category**, for example: Superscript, Fraction, Quote, etc.
```

Glyph name extension

> : sort by **extension**, for example **.case**, **.numerator**, etc.

Reviewer: ```suggestion
: Group by **glyph name extension**, for example **.case**, **.numerator**, etc.
```

<div class="alert alert-primary mt-3" role="alert" markdown='1'>
#### Pro tip: 
{: .alert-heading}
- Try: Group by **Category** + **Sub-Category**.
{: .mb-0 }
</div>

- - -

##### Project glyph sets

Project specific glyph sets (activate one or many via checkboxes).

This font's glyphs
: shows **all glyphs** of the font project

<img height="30" src='{{ site.url }}/images/icons/plus.svg' />*
: add a new glyph set to this specific font project

- - -

##### My glyph sets

> Fontra specific glyph sets (activate one or many via checkboxes).

Reviewer: It's not "Fontra specific", it is stored "for the user", in contrast as "withg the font data" aka "in the font project".

Author: Changed to `User specific glyph sets`

> User specific glyph sets (activate one or many via checkboxes).

Reviewer: ```suggestion
User-specific glyph sets (activate one or many via checkboxes). These are not stored in the font, but locally for the user.
```
